### PR TITLE
Removes `Confirm Your Transaction` modal for lightwallet users 

### PR DESCRIPTION
**Pull request by Moejoe90**
This adds a Loading Indicator but creates a bad modal size change effect
That problem should be handled here 
https://github.com/Neufund/platform-frontend/issues/2522

### CODE REVIEW

File: app/components/modals/tx-sender/TxSender.tsx [hunk 1]
```diff
@@ -183,7 +187,14 @@ const TxSenderBody: React.FunctionComponent<Props> = ({
       );
 
     case ETxSenderState.SIGNING:
-      return <SigningMessage />;
+      return walletType && walletType === EWalletType.LIGHT ? (
```
Comment (desfero) [inline app/components/modals/tx-sender/TxSender.tsx]:
```suggestion
      return walletType === EWalletType.LIGHT ? (
```

File: app/modules/web3/selectors.spec.ts [hunk 2]
```diff
@@ -45,54 +46,62 @@ describe("web3 > selectors", () => {
...
+          previousConnectedWallet: {
+            walletType: EWalletType.LIGHT,
+            walletSubType: EWalletSubType.UNKNOWN,
+            address: dummyEthereumAddress,
+            email,
+            salt,
+          },
         },
-      };
+      } as IAppState;
```
Comment (desfero) [inline app/modules/web3/selectors.spec.ts]: please use `Partial<IAppState>` then it's easier to catch issues related to store shape change

File: app/modules/web3/selectors.spec.ts [hunk 3]
```diff
@@ -45,54 +46,62 @@ describe("web3 > selectors", () => {
...
-        isUnlocked: false,
-        wallet: getDummyLedgerWalletMetadata(),
-      };
+      const state = {
+        web3: {
+          connected: true,
+          isUnlocked: false,
+          wallet: getDummyLedgerWalletMetadata(),
+        },
+      } as IAppState;
```
Comment (desfero) [inline app/modules/web3/selectors.spec.ts]: please use `Partial<IAppState>` then it's easier to catch issues related to store shape change

File: app/modules/web3/selectors.spec.ts [hunk 4]
```diff
@@ -45,54 +46,62 @@ describe("web3 > selectors", () => {
...
-        isUnlocked: false,
-        wallet: getDummyBrowserWalletMetadata(),
-      };
+      const state = {
+        web3: {
+          connected: true,
+          isUnlocked: false,
+          wallet: getDummyBrowserWalletMetadata(),
+        },
+      } as IAppState;
```
Comment (desfero) [inline app/modules/web3/selectors.spec.ts]: please use Partial<IAppState> then it's easier to catch issues related to store shape change

File: app/modules/web3/selectors.spec.ts [hunk 5]
```diff
@@ -45,54 +46,62 @@ describe("web3 > selectors", () => {
...
-        isUnlocked: false,
-        wallet: getDummyLightWalletMetadata(),
-      };
+      const state = {
+        web3: {
+          connected: true,
+          isUnlocked: false,
+          wallet: getDummyLightWalletMetadata(),
+        },
+      } as IAppState;
```
Comment (desfero) [inline app/modules/web3/selectors.spec.ts]: please use `Partial<IAppState>` then it's easier to catch issues related to store shape change

File: app/modules/web3/selectors.ts [hunk 6]
```diff
@@ -54,7 +54,7 @@ export const selectIsLightWallet = (state: IWeb3State): boolean => {
   );
 };
 
-export const selectIsExternalWallet = (state: IWeb3State): boolean => {
+export const selectIsExternalWallet = (state: IAppState): boolean => {
```
Comment (desfero) [inline app/modules/web3/selectors.ts]: 🎖

File: app/modules/web3/selectors.spec.ts (hunk 2 above)
Comment (Moejoe90) [reply app/modules/web3/selectors.spec.ts]: The problem then when you make `IAppState` `Partial<IAppState>` is it won't fulfil the type

**Pull request closed without merging**